images/hypershift.yml

```diff
@@ -0,0 +1,26 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/hypershift
+content:
+  source:
+    dockerfile: Dockerfile.control-plane
+    git:
+      branch:
+        target: main
```

Comment:
just waiting for the creation of `release-4.10` branch upstream

Reply:
well, feature freeze is approaching, we can just merge this as-is and change it later, once they manage to create the branches.